### DISCUSSION TITLE
ODBC drivers - can't download from repository

### DISCUSSION THREAD

**Discussion by cs-pady** [dbeaver/dbeaver#10773] [Q&A] [answered]
#### System information: 
- Windows 7 SP1 (64 bit)
- DBeaver version: 3.8.1 (Enterprise Edition), and 7.2.5 (Community Edition)

#### Connection specification:
- Unsure, IBM DB2 I think
- Driver name: ODBC
- Do you use tunnels or proxies (SSH, SOCKS, etc)? : No

#### Describe the problem you're observing: 

I am trying to connect to a database using ODBC drivers, but I can't download the drivers from repository. Drivers in question:

repo:/drivers/odbc/jdbc-odbc-bridge-jre7.jar
repo:/drivers/odbc/x64/JdbcOdbc.dll

Error is simply:

'Driver file download failed. 
Do you want to retry?

Reason:dbeaver.jkiss.org'

I am using dbeaver on a work machine so the EE version cannot be updated beyond what is available internally from my IT. I downloaded to the most recent CE (7.2.5) and I get the same issue as in EE.

_Is it possible to make drivers available from download on GitHub to circumvent the download issue?_

#### Steps to reproduce, if exist:

1. New Connection
2. Select ODBC, click Next
3. enter Database/Schema (e.g. DSNXX)
4. enter username + password, click Test Connection
5. Window pops up prompting me to download aforementioned files.
6. Click Download
7. Get Error as per above

#### Include any warning/errors/backtraces from the logs

N/A - nothing in error log for this.

**Comment by kseniiaguzeeva** [comment]
Do you have Proxy or Firewall?
And please send log files https://github.com/dbeaver/dbeaver/wiki/Log-files
I can also offer you Enterprise Edition https://dbeaver.com/ (Free trial period is available). ODBC is already in DBeaver there.

**Comment by cs-pady** [comment]
Hi @kseniiaguzeeva , thanks for replying!

As this is a work machine it may well be a firewall issue, which means downloading the requisite files outside of dbeaver and adding to the ODBC driver manually might be the best route...what do you think?

My logs don't show anything with regards to the driver download issue :(

**Comment by kseniiaguzeeva** [comment, marked as answer]
Yes, you can. Please try:
https://dbeaver.io/files/jdbc/drivers/odbc/x64/JdbcOdbc.dll
https://dbeaver.io/files/jdbc/drivers/odbc/jdbc-odbc-bridge-jre7.jar

**Comment by cs-pady** [comment]
Thank you very much @kseniiaguzeeva ! That has worked and I am now connected to the DB.

Many thanks and happy holidays!

**Comment by kseniiaguzeeva** [comment]
Nice to hear!
